L1Trigger/TrackFindingTracklet/interface/DuplicateRemoval.h

```diff
@@ -9,42 +9,39 @@
 
 namespace trklet {
 
-  /*! \class  trklet::DR
+  /*! \class  trklet::DuplicateRemoval
    *  \brief  Class to bit- and clock-accurate emulate duplicate removal
    *          DR identifies duplicates based on pairs of tracks that share stubs in at least 3 layers.
    *          It keeps the first such track in each pair.
```

Comment:
Modify this comment to make clear that the track order is determined by TrackMultiplexer.

Reply:
done

Reply:
Modify comment to mention ProducerTM, since it's not obvious that the Track Multiplexer can be found there.

Reply:
done